packages/web-react/src/components/Tabs/useTabsStyleProps.ts

```diff
@@ -23,6 +31,19 @@ export function useTabsStyleProps(props: SpiritTabsProps = { selectedId: '', for
   const tabsPaneClass = `${tabsClass}Pane`;
   const tabsSelectedClass = 'is-selected';
 
+  const tabsStyle: TabsCSSProperties = {};
+
+  if (typeof spacing === 'object' && spacing !== null) {
```

Comment:
I would like to leave here and in `useStackStyleProps` some kind of comment that suggests refactoring this block of code to a function and sharing it when we want to copy this for a third time.

I am ok now with this approach but when we want to add a `spacing` prop to another component there should be a note that it is time to refactor it :-)

Reply:
Refactored. New hook `useSpacingStyle` added (including UT). Refactored also for Stack component.